src/main/java/machine_learning/classifiers/ensembles/voting/stacking/AbstractStacking.java

```diff
@@ -117,9 +117,9 @@ protected void initInstances() {
         double[][] dists = new double[modules.length][];
         
         for(int m = 0; m < modules.length; m++){
-            long startTime = System.currentTimeMillis();
+            long startTime = System.nanoTime();
```

Comment:
May be worth adding a time unit para onto storeModuleTestResult I guess, but really if everything just uses nanos it's not too important 

Would be able to get rid of this todo as well :P 

https://github.com/uea-machine-learning/tsml/blob/master/src/main/java/experiments/Experiments.java#L515

Reply:
I think one of the big issues last time we talked about a complete switch over was old results files, but probably easier to just convert those files when loading at this point rather than allow both. 

Don't know if you can just assume any file which doesn't state its using nanos is milliseconds.

Reply:
I think we can go fully nano, old files can be rerun